code/drasil-website/lib/Drasil/Website/Example.hs

```diff
@@ -183,6 +185,17 @@ getCodeRef ex@E{sysInfoE=SI{_sys = sys}, choicesE = chcs} l verName =
     -- Program language converted for use in file folder navigation.
     programLang = convertLang l
 
+-- | Similar to 'getCodeRef', but gets the source code references 
+-- and uses 'getSourceCodePath' instead.
+getSourceCodeRef :: Example -> Reference
+getSourceCodeRef ex@E{sysInfoE=SI{_sys = sys}} = 
```

Comment:
`buildDrasilSrcRef` might be a _slightly_ better name. `get Source Code Ref`  tells me that we're going to be getting some sort of information _from_ `Example` rather than building something using it.

Reply:
Alt.: `buildDrasilExSrcRef` or `buildDrasilExampleSrcRef`.